src/client/content_cao.cpp

```diff
@@ -1672,22 +1715,14 @@ void GenericCAO::processMessage(const std::string &data)
 			int rating = readS16(is);
 			m_armor_groups[name] = rating;
 		}
-	} else if (cmd == AO_CMD_UPDATE_NAMETAG_ATTRIBUTES) {
-		// Deprecated, for backwards compatibility only.
```

Comment:
was this sent by any post-5.0 versions? Which ones?

Reply:
Yes, all of them.
The reason that this can just be removed is that that the properties already include the nametag data.